tests/test_build_latex.py

```diff
@@ -344,6 +355,11 @@ def test_numref_with_language_ja(app, status, warning):
     assert ('\\hyperref[\\detokenize{foo:foo}]{Sect.\\ref{\\detokenize{foo:foo}} '
             '\\nameref{\\detokenize{foo:foo}}}') in result
 
+    # sphinxmessage.sty
+    result = (app.outdir / 'sphinxmessage.sty').text(encoding='utf8')
+    print(result)
+    assert r'\renewcommand{\figurename}{図}' in result
```

Comment:
OMG, this does not confirm the `\renewcommand` is *not* surrounded by `\captions...`.